.github/workflows/ci.yaml

```diff
@@ -681,7 +681,7 @@ jobs:
       - name: Run codegen to pure_dart example (non-windows)
         if: ${{ matrix.os.family != 'windows' }}
         working-directory: ./frb_codegen
-        run: cargo run --package flutter_rust_bridge_codegen --bin flutter_rust_bridge_codegen --
+        run: cargo run --package flutter_rust_bridge_codegen --bin flutter_rust_bridge_codegen --features "chrono,uuid" --
           --rust-input ../frb_example/pure_dart/rust/src/api.rs
```

Comment:
btw dart web tests seem to fail (unrelated to this line of code)

Reply:
it says timeout so not sure : github ci has an option to rerun only failed jobs.

Reply:
Ok, will not be problem if future commit it is green :)